srcpkgs/calamares/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'calamares'
+pkgname=calamares
+version=3.2.44.2
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release
+                -DCMAKE_INSTALL_PREFIX=/usr
+                -DCMAKE_INSTALL_LIBDIR=lib
+                -DWITH_PYTHONQT:BOOL=ON
+                -DINSTALL_CONFIG:BOOL=ON
+                -DINSTALL_POLKIT:BOOL=ON
+                -DBoost_NO_BOOST_CMAKE=ON
+                -DSKIP_MODULES='webview interactiveterminal services-systemd dummyprocess dummypython dummycpp dummypythonqt'"
```

Comment:
Doesn't work, this need to be separated by semicolons, not spaces to be a single argument. I'm not sure why this didn't break build.